source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,200 @@
+
+
+Setting up Unbound
+------------------
+
+Unbound is a powerful validating, recursive, caching DNS resolver. It’s used by some of the biggest tech companies in the world as well as home users, who use it together with ad blockers and firewalls, or self-run resolvers. Setting it up for your home network can be quite simple as we’ll showcase below.
+
+Setting up your own DNS resolver for your entire home network requires a couple of things. Namely, a recursive DNS resolver (who knew!), and a dedicated machine where the resolver runs, which is always on and accessible to the entire network. This can be as simple as a Raspberry Pi, or any other machine that is always online, connected to your home router.
+
+Because of the variety of machines that Unbound can run on we cannot create a comprehensive tutorial for all possible options. For the extent of this tutorial we will use :command:`Ubuntu 20.04.1 LTS` as a stepping stone which you could use and adapt for other machines.
+
+While you could download the code from Github and build it yourself, getting a copy can be as simple as running:
+
+.. code-block:: bash
+
+	sudo apt-get update
+	sudo apt install unbound -y
+
+This gives you a full, compiled, and running version of Unbound which behaves as a caching recursive DNS resolver out of the box for the local machine. 
+
+.. after it has been written, link to the local-stub to show how to compile and build.
+
+Do note that the current setup is only reachable on this machine.
+
+Testing the server locally
 --------------------------
 
-Using Unbound as the resolver for a home network.
+To verify that the server works correctly it’s a good idea to test it before committing the entire network to it. Luckily we can test this on the machine that you installed Unbound on (locally) and from any other machine (remotely) that will be using the resolver after we expose Unbound to the network.
+
+The command for local testing is:
+
+.. code-block:: bash
+
+	dig example.com @127.0.0.1
+
+Here we tell the :command:`dig` tool to look up the IP address for example.com, and to ask this information to the server running at the IP address ``127.0.0.1``, which is where our Unbound machine is running by default.
+We can verify that Unbound has indeed answered our query instead of the default resolver that is present on Ubuntu by default. In the output of every :command:`dig` command there is ``ANSWER SECTION`` which specifies the server which has answer the query under ``SERVER`` entry. The entry should be:
+
+.. code-block:: bash
+
+	;; SERVER: 127.0.0.1#53(127.0.0.1)
+
+In the next section we will be disabling the default Ubuntu resolver. To verify that we do it correctly there it is useful to know the address of the default resolver as a baseline. For this baseline, we also use a :command:`dig` query but time without specifying an IP address which the uses the machines default DNS resolver.
```

Comment:
s/do it correctly there it is/do it correctly it is/
s/For this baseline, we also use a dig query but time without/For this baseline we also use a dig query, but this time without/

"without specifying an IP address which the uses the machines default DNS resolver" - this should be "which uses the" instead of "which the uses the". However I would also put the "which uses the machines default DNS resolver" in parentheses and word it slightly differently, e.g.:

For this baseline we also use a :command:`dig` query, but this time without specifying an IP address (which causes dig to use the machines default DNS resolver).

I believe for absolute correctness there should also be an apostrophe so that you write "machine's default DNS resolver" to denote ownership rather than plurality.